### DISCUSSION TITLE
How to use --multiuser ?

### DISCUSSION THREAD

**Discussion by AB0x** [LostRuins/koboldcpp#1100] [Q&A]
how to use this feature with example.

**Comment by LostRuins** [comment]
This controls the queue length allowed by the server. Queue length means how many requests are allowed to wait in turn before the server stops accepting new requests.

`--multiuser 10` allows for keeping up to 10 users in the queue

`--multiuser 0` disables queue completely, meaning only 1 request is allowed at any time.
